### ISSUE TITLE
User interface: use GUI Helper services to make dialogs more consistent with NVDA Core (requires NVDA 2016.4 or later)

### ISSUE THREAD

**Issue by josephsl** [NVDARemote/NVDARemote#92]
Hi,

In NVDA 2016.4, GUI helper services (gui.guiHelper) is now available to simplify GUI code creation and maintenance. Also, add-on authors doesn't have to worry about spacing and having to use two step GUI creation for combo boxes and related controls (one can use guiHelper.BoxSizerHelper.addLabeledControl for this). Thus I suggest using GUI Helper for add-on UI (and to make the UI consistent with NVDA Core itself).

Thanks.

**Comment by leonardder** [comment]
How are add-ons which use guiHelper supposed to handle compatibility with older NVDA builds? Is it even desirable to break this compatibilityy,?

N.B. Sent on behalf of @BabbageCom

**Comment by josephsl** [comment]
Hi,

One approach is using a if/else compatibility layer that uses a global variable (defined at the top of the file).
Thanks.
